x-pack/plugins/ml/server/models/job_service/new_job_caps/new_job_caps.ts

```diff
@@ -22,6 +23,14 @@ export function newJobCapsProvider(client: IScopedClusterClient) {
     const { aggs, fields } = await fieldService.getData();
     convertForStringify(aggs, fields);
 
+    // Remove the _doc_count field as we don't want to display this in the fields lists in the UI
+    const docCountIndex = fields.findIndex((f: Field) => {
```

Comment:
I think using a filter here would be faster.
```
const fieldsWithoutDocCount = fields.filter(({id}) => id !== _DOC_COUNT)
```

Reply:
Done in https://github.com/elastic/kibana/pull/85444/commits/f1f8edf4631a24d2f917c34952f83b165e67fe5c